plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java

```diff
@@ -198,12 +198,9 @@ public void testSchemaOperations()
         assertUpdate("CREATE SCHEMA new_schema");
         assertUpdate("CREATE TABLE new_schema.test (x bigint)");
 
-        try {
-            getQueryRunner().execute("DROP SCHEMA new_schema");
-            fail("Should not be able to drop non-empty schema");
-        }
-        catch (RuntimeException e) {
-        }
+        assertThatThrownBy(() -> getQueryRunner().execute("DROP SCHEMA new_schema"))
```

Comment:
Can you please also fix `io.trino.plugin.phoenix.TestPhoenixConnectorTest#testSchemaOperations`? It is a copy.

Reply:
@kokosing 
`io.trino.plugin.phoenix.TestPhoenixConnectorTest#testSchemaOperations` has been fixed in https://github.com/trinodb/trino/pull/6767.

https://github.com/trinodb/trino/blob/master/plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java#L195-L207